### PR TITLE
Remove CentOS guide that was already deprecated a while ago

### PR DESCRIPTION
**Pull request by errordeveloper**
As we've discussed between sig-cluster-lifecycle and sig-docs a few weeks ago, there are getting started guides that are evidently out of date and will need to be updated and pass conformance.

In the process of identifying guide owners and maintainers, it was evident that this CentOS guide can be removed safely.

cc @lavalamp @thockin @chenopis

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/7242)

### CODE REVIEW

**Comment by lavalamp** [comment]
Gave the tech lgtm label. 1.1 setup guide is definitely not useful at this point.

File: _redirects [hunk 1]
```diff
@@ -179,6 +179,9 @@
 /docs/getting-started-guides/vagrant/     /docs/getting-started-guides/alternatives/ 301
 /docs/getting-started-guides/windows/While/     /docs/getting-started-guides/windows/ 301
 
+/docs/getting-started-guides/centos/* /docs/setup/independent/create-cluster-kubeadm/ 301
+/cn/docs/getting-started-guides/centos/* /docs/setup/independent/create-cluster-kubeadm/ 301
```
Comment (xiangpengzhao) [inline _redirects]: cn docs are not actively maintained in this repo, so we don't need this line I think. cc @markthink @hanjiayao for confirmation.

Comment (errordeveloper) [reply _redirects]: I've remove it.

**Pull request merged**